utils/anautilities.py

```diff
@@ -875,6 +875,32 @@ def parseCalFile(filename=None):
 
     return (calDAC2Q_m, calDAC2Q_b)
 
+def parseArmDacCalFile(filename):
+    import numpy as np
+    import root_numpy as rp #note need root_numpy-4.7.2 (may need to run 'pip install root_numpy --upgrade')
+    import ROOT as r
+
+    # Set the CAL DAC to fC conversion
+    coef4 = np.zeros(24)
+    coef3 = np.zeros(24)
+    coef2 = np.zeros(24)
+    coef1 = np.zeros(24)
+    coef0 = np.zeros(24)
+    list_bNames = ["vfatN","coef4","coef3","coef2","coef1","coef0"]
+    calTree = r.TTree('calTree','Tree holding VFAT Calibration Info')
+    calTree.ReadFile(filename)
```

Comment:
Also can you add this line `891` in a `try...except IOError` block with a `printRed` statement that says if `IOError` occurs the file is not readable or does not exist, e.g.:

```python
try:
    calTree.ReadFile(filename)
except IOError:
    printRed("file {0} is not readable or does not exist, please cross-check".format(filename))
    exit(os.EX_IOERR)
```

Reply:
done